<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeacherExamManagement</class>
 <widget class="QDialog" name="TeacherExamManagement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Assigned Exams</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="searchBar">
     <property name="placeholderText">
      <string>Search Exam...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="examTable"/>
   </item>
   <item>
    <widget class="QPushButton" name="closeButton">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
